src/pudl_archiver/archivers/classes.py

```diff
@@ -758,6 +759,18 @@ async def download_all_resources(
                         ]
                     )
 
+                    # Check if fail_fast is set and there are failed file level validations
```

Comment:
After every new resource is downloaded, we will check to see if any file-level validation tests have failed (these check for empty/invalid files based on the filetype extension), and if any of the tests fail and `fail_fast` is set it will immediately raise an error. This means that if we download a file and find that it's corrupted, we will fail before it overwrites the existing file in GCS.

Reply:
How do we handle the exception that's raised? And what behavior do we want here?

* Will the entire archiving run fail halfway through, leaving some EQR files updated, but others not?
* Will it bail on this file and attempt to update the rest of the files?
* Will it retry this file and also continue attempting to update the other files?

In the case of the EQR, my inclination is that it should retry, and even if it can't get this file updated because it's corrupted or something, it should still attempt to update the other files, so that at least on a per-file basis, we have the freshest available data, even if one of them got messed up.